### PR TITLE
Move 2

### PR DESCRIPTION
**Pull request by danf-iolap1**
Cleared B3

### CODE REVIEW

**Comment by mentor-tara** [comment]
### Oh no! You hit a mine. Game over :slightly_frowning_face:

This is the current board:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** |   |   |   |   |   |   |   |   |
| **2** |   |   |   |   |   |   |   |   |
| **3** | 1 | :bomb: |   |   |   |   |   |   |
| **4** |   |   |   |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |
| **6** |   |   |   |   |   |   |   |   |
| **7** |   |   |   |   |   |   |   |   |
| **8** |   |   |   |   |   |   |   |   |

And this is the revealed board:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** | 1 | :bomb: | 2 | 1 | 0 | 0 | 0 | 0 |
| **2** | 2 | 3 | :bomb: | 1 | 1 | 1 | 1 | 0 |
| **3** | 1 | :bomb: | 2 | 2 | 2 | :bomb: | 3 | 2 |
| **4** | 2 | 2 | 2 | 1 | :bomb: | 4 | :bomb: | :bomb: |
| **5** | 2 | :bomb: | 1 | 1 | 2 | :bomb: | 3 | 2 |
| **6** | :bomb: | 2 | 1 | 0 | 1 | 1 | 1 | 0 |
| **7** | 1 | 1 | 0 | 0 | 0 | 0 | 0 | 0 |
| **8** | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |

---

### This game is over but I created a PR so you can start a new game. You can close this PR.[](clear-b3)

**Review by danf-iolap1 — COMMENTED** [review]
yup, looks good

File: commands.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "commands": [
-     "clear A3"
+     "clear A3",
+     "clear b3"
```
Comment (danf-iolap1) [inline commands.json:4]: yup, this is right

**Review by danf-iolap1 — COMMENTED** [review]
good

**Pull request closed without merging**